com.unity.perception/Runtime/GroundTruth/RenderedObjectInfoGenerator.cs

```diff
@@ -47,9 +47,11 @@ public void Execute()
                     };
                     for (var i = 0; i < rowSlice.Length; i++)
                     {
-                        var value = rowSlice[i];
+                        var packed = InstanceIdToColorMapping.GetPackedColorFromColor(rowSlice[i]);
```

Comment:
So the main reason is that Color32 doesn't really have any of the methods that you would hope that it would have out of the box, such as any comparison operators (=,>,<), and assignment operators, or hashing methods. Also, in early experiments I had some crashing issues, but I'm not sure what was the actual cause of that.

Reply:
That's really annoying. ok, makes sense.